vita3k/gui/src/app_context_menu.cpp

```diff
@@ -342,7 +368,9 @@ void draw_app_context_menu(GuiState &gui, HostState &host, const std::string &ap
                     open_path(SHADER_LOG_PATH.string());
                 ImGui::EndMenu();
             }
-            if (!host.cfg.show_live_area_screen && ImGui::BeginMenu("Live Area")) {
+            if (ImGui::BeginMenu("Live Area")) {
+                if (ImGui::MenuItem(gui.lang.live_area[START].c_str()))
```

Comment:
use string literal here
```suggestion
                if (ImGui::MenuItem("%s", gui.lang.live_area[START].c_str()))
```

Reply:
oh i have forget this pr, i have hurt for corect solution lol